.travis.yml

```diff
@@ -1,12 +1,26 @@
 language: node_js
+
 node_js:
   - '7'
 
+addons:
+  apt:
+    sources:
+    - google-chrome
+    packages:
+    - google-chrome-stable
+
+before_install:
+  - export CHROME_BIN=/usr/bin/google-chrome
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
```

Comment:
I've never used Chrome in travis. Cool to see the config. I guess I've usually used phantomjs. Idk what is suggested for Angular 2. I'm fine with Chrome :)